test/evergreen.yml

```diff
@@ -1015,90 +1026,80 @@ tasks:
       - name: compile
     commands:
       - func: "fetch artifacts"
-      - command: shell.exec
-        params:
-          working_dir: "wiredtiger/build_posix/"
-          script: |
-            set -o errexit
-            set -o verbose
-
-            ${test_env_vars|} $(pwd)/test/cppsuite/run -t base_test -C 'debug_mode=(cursor_copy=true)' -f test/cppsuite/configs/base_test_default.txt -l 2
+      - func: "cppsuite test"
+        vars:
+          test_name: base_test
+          test_config: debug_mode=(cursor_copy=true)
+          test_config_filename: base_test_default.txt
 
   - name: cppsuite-hs-cleanup-default
     tags: ["pull_request"]
     depends_on:
       - name: compile
     commands:
       - func: "fetch artifacts"
-      - command: shell.exec
-        params:
-          working_dir: "wiredtiger/build_posix/"
-          script: |
-            set -o errexit
-            set -o verbose
-
-            ${test_env_vars|} $(pwd)/test/cppsuite/run -t hs_cleanup -C 'debug_mode=(cursor_copy=true)' -f test/cppsuite/configs/hs_cleanup_default.txt -l 2
+      - func: "cppsuite test"
+        vars:
+          test_name: hs_cleanup
+          test_config_filename: hs_cleanup_default.txt
+          test_config: debug_mode=(cursor_copy=true)
```

Comment:
Just a nit - can we keep the sequence of setting `test_config_filename` and `test_config_filename` consistent across tasks?

Reply:
I have alphabetized all arguments in 6df99a432fe1dccf4c34f341fe40682076ea3d11